identity/src/main/java/com/stripe/android/identity/states/MBDetector.kt

```diff
@@ -0,0 +1,232 @@
+package com.stripe.android.identity.states
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Rect
+import androidx.annotation.IntegerRes
+import com.microblink.capture.CaptureSDK
+import com.microblink.capture.analysis.FrameAnalysisResult
+import com.microblink.capture.analysis.FrameAnalysisStatus
+import com.microblink.capture.directapi.AnalysisError
+import com.microblink.capture.directapi.AnalyzerRunner
+import com.microblink.capture.directapi.FrameAnalysisResultListener
+import com.microblink.capture.image.ImageRotation
+import com.microblink.capture.image.InputImage
+import com.microblink.capture.result.Side
+import com.microblink.capture.settings.AnalyzerSettings
+import com.microblink.capture.settings.BlurPolicy
+import com.microblink.capture.settings.CaptureStrategy
+import com.microblink.capture.settings.GlarePolicy
+import com.microblink.capture.settings.LightingThresholds
+import com.microblink.capture.settings.TiltPolicy
+import com.stripe.android.identity.R
+import com.stripe.android.identity.analytics.IdentityAnalyticsRequestFactory
+import com.stripe.android.identity.ml.AnalyzerInput
+import com.stripe.android.identity.networking.IdentityRepository
+import kotlin.coroutines.resume
+import kotlin.coroutines.suspendCoroutine
+import com.stripe.android.identity.networking.models.VerificationPageStaticContentDocumentCaptureMBSettings as MBSettings
+
+internal class MBDetector private constructor(settings: MBSettings) {
+    enum class CaptureFeedback(
+        @IntegerRes val stringResource: Int?
+    ) {
+        DOCUMENT_FRAMING_NO_DOCUMENT(null),
```

Comment:
Do we not have an error message for this case?

Reply:
No, this is an expected state, when user just turn on the camera and doesn't point to a document yet.
In this case the feedback(the null param here) is not from MBDetector, but an initial string resource to tell the user to point to the camera